src/dataplane-protocol/src/batch.rs

```diff
@@ -15,10 +15,7 @@ use crate::derive::Encode;
 
```

Comment:
need to bump up minor version and make sure all dependent crates use new version

Reply:
Bumped version to 0.5.1